orders/src/main/java/com/orders/crudyorders/model/Order.java

```diff
@@ -0,0 +1,83 @@
+package com.orders.crudyorders.model;
+
+import javax.persistence.*;
+
+public class Order
+{
```

Comment:
Make sure to add the `@Entity` and `@Table` (name must be set as well; it is advisable to use plural casing to prevent naming conflicts - especially important here because "order" is an SQL keyword that can break database operations if used as a table name) annotations on top of your class.

Example:
```
@Entity
@Table(name = "orders")
public class Order
{
```